### ISSUE TITLE
Regarding Update Graph

### ISSUE THREAD

**Issue by teraiyamayurcontact** [highcharts/highcharts-android#229]
HIXAxis hixAxis = new HIXAxis();

i am working on Android sdk. when i update Xaxis and Yaxis second time according to filter data in app graph not getting update.

Thanks in advance

**Comment by teraiyamayurcontact** [comment]
Thanks for your replay.

Once graph loaded then we are filter data and second time we have set new data in option and but its not get reflected.

    hiChartTrend.redraw();
    hiChartTrend.refreshDrawableState();
    hiChartTrend.resetPivot();

i have tried above method for after select filter from dropdown data not getting update.

Thanks

**Comment by soommy12** [comment]
If you are setting new data I guess you're speaking of the new series set not the x-axis. If you have new series for your chart, you can use the `setSeries(newArrayListOfSeries)` method. That should refresh the chart (the redraw, reload, and so on are not needed). Also, if that won't work for any reason try to reassign the options object which contains a new data set (`chartview.setOptions(newOptions)`).

**Comment by teraiyamayurcontact** [comment]
Thanks for your replay.

chartview.setOptions(newOptions) this one already done but not working setSeries(newArrayListOfSeries) let me try again with this.

**Comment by MikolajMichalczak** [comment]
Closing due to inactivity. Please feel free to reopen the ticket if you have any further questions or if the problem persists.
